lib/dynamic-bootstrap.js

```diff
@@ -9,9 +9,9 @@ var path = require('path'),
     Q = require('q'),
```

Comment:
since we're converting the file to ES6/7, we probably want to convert all these old requires to imports